### PR TITLE
Added support for handling boards without builtin speakers

### PR DESCRIPTION
**Pull request by brentru**
Uses tone() from simpleio to avoid using builtin speakers on boards without them (square wave instead of sine wave for a piezo) 

@ladyada

### CODE REVIEW

**Comment by ladyada** [comment]
looks good, please refactor to avoid the big code dupliation!

```
+            p = None
+            d = duration
+            if note[0].isdigit():
+                d = int(note[0])
+                p = note[1]
+            else:
+                p = note[0]
+            if "." in note:
+                d *= 1.5
+            if "#" in note:
+                p += "#"
+            o = octave
+            if note[-1].isdigit():
+                o = note[-1]
+            p = o + p
```

etc

File: adafruit_rtttl.py [hunk 1]
```diff
@@ -61,6 +62,8 @@
          "7c" : 2093,
          "7c#": 2217.5}
 
+def
```
Comment (tannewt) [inline adafruit_rtttl.py:65]: Why is this here?

**Comment by tannewt** [comment]
There are outstanding comments and no response. Please reopen if you want to continue this.
